client/gutenberg/extensions/markdown/editor.scss

```diff
@@ -1,10 +1,139 @@
+// @TODO: Replace with Gutenberg variables
```

Comment:
I'm missing these also for Tiled Gallery block and we'll likely have more.

I could be confusing to just import Gutenberg vars here since it would be impossible to tell from names where they're coming from and there might be collisions.

We already have these in Calypso, here:

[`client/gutenberg/editor/edit-post/assets/stylesheets/_colors.scss`](https://github.com/Automattic/wp-calypso/blob/47b41fb386aa29cd832805dbaaa600cd2f786ad4/client/gutenberg/editor/edit-post/assets/stylesheets/_colors.scss)

[`client/gutenberg/editor/edit-post/assets/stylesheets/_variables.scss`](https://github.com/Automattic/wp-calypso/blob/47b41fb386aa29cd832805dbaaa600cd2f786ad4/client/gutenberg/editor/edit-post/assets/stylesheets/_variables.scss)

Should we just import those or perhaps create a wrapper function like `gutenberg( 'editor-html-font' )` ?

Reply:
Sounds like something we could play with.